Source/WebKit/UIProcess/Launcher/playstation/ProcessLauncherPlayStation.cpp

```diff
@@ -80,7 +80,7 @@ void ProcessLauncher::launchProcess()
     };
 
 #if USE(WPE_BACKEND_PLAYSTATION)
-    auto appLocalPid = ProcessProviderLibWPE::singleton().launchProcess(m_launchOptions, argv, socketPair.client);
+    auto appLocalPid = ProcessProviderLibWPE::singleton().launchProcess(m_launchOptions, argv, socketPair.client.release());
```

Comment:
Oops, it doesn't take ownership of the fd. (Prior to my commit that broke the build, there was a close() call down below here in `ProcessLauncher::launchProcess`.)

```suggestion
    auto appLocalPid = ProcessProviderLibWPE::singleton().launchProcess(m_launchOptions, argv, socketPair.client.value());
```